### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by graceuche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# The Final Project for Northeaster University Accelerated Data Analytics Professional Certification Program
+
```
Comment (missKatiaPunter) [inline README.md:2]: Add a paragraph text that explains what Northeastern is and what criteria the project had. You can copy and paste from Canvas or write it in your own words

**Review by missKatiaPunter — CHANGES_REQUESTED** [review]

**Review by missKatiaPunter — APPROVED** [review]
Well done, @graceuche 👍👏🏻👏🏻👏🏻👏🏻👏🏻

**Comment by graceuche** [comment]
> Well done, @graceuche 👍👏🏻👏🏻👏🏻👏🏻👏🏻

Thanks for your feedback, Dr. Katia

**Comment by graceuche** [comment]
Thanks Dr.Katia.
I appreciate your feedback.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**